api/src/main/java/org/pentaho/hadoop/shim/HadoopConfigurationLocator.java

```diff
@@ -34,6 +34,7 @@
 import java.util.Map.Entry;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.apache.commons.lang.StringUtils;
```

Comment:
@ArtsiomYurhevich Could you please update pull-request to not use StringUtils from new library. Try to use kettle core 
org.pentaho.di.core.util.StringUtil or just check for empty directly. Also can you send pull-request to master with such update. Thanks.